routes/conversation.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const router = express.Router();
+const Invitation = require('../models/Invitation');
+const Conversation = require('../models/Conversation');
+const User = require('../models/User');
+const auth = require('../middleware/auth');
+
+router.post('/:members', auth, async (req, res) => {
+  const memberEmails = req.params.members.split('&');
+  const membersFromDb = [];
+
+  try {
+    for (const email of memberEmails) {
+      const foundMember = await User.findOne({ email });
+      if (foundMember) membersFromDb.push(foundMember);
+    }
```

Comment:
Should we return an error if any are not found?

Reply:
I think I'm going to delete this now as Bonnie has a better implementation in one of her branches.

Reply:
if (foundMember) membersFromDb.push(foundMember);
else return res.status(404).json({ msg: `User ${email} not found` });

Reply:
is this the route you were mentioning about creating a conversation? if so, yeah we don't need a controller for this, can be done later on via sockets